server/routes/induction/update/qualificationLevelUpdateController.test.ts

```diff
@@ -54,6 +54,10 @@ describe('qualificationLevelUpdateController', () => {
       const inductionDto = aLongQuestionSetInductionDto()
       req.session.inductionDto = inductionDto
       req.session.qualificationLevelForm = undefined
+      req.session.pageFlowQueue = {
+        pageUrls: [`/prisoners/${prisonNumber}/induction/qualifications`],
```

Comment:
Should this have at least 1 elements in it? and the currentPageIndex is zero indexed

Reply:
I've changed this to:

```
pageUrls: [
   `/prisoners/${prisonNumber}/induction/qualifications`,
   `/prisoners/${prisonNumber}/induction/qualification-level`,
],
```

But I think the index needs to remain at 1, since the `routerRequestHandler` is not involved here.